src/averageAges.js

```diff
@@ -37,7 +42,16 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = withChildren
+    ? people.filter((p) => p.sex === 'f'
+      && people.find((p1) => p1.mother === p.name))
+    : people.filter((p) => p.sex === 'f');
```

Comment:
Fix formatting according to the [checklist](https://github.com/mate-academy/js_average-ages_test-task/blob/master/checklist.md).
```suggestion
  const women = withChildren
    ? people.filter(
        p => p.sex === 'f' && people.find(p1 => p1.mother === p.name)
      )
    : people.filter(p => p.sex === 'f');
```